src/Makefile.vc6

```diff
@@ -182,6 +182,23 @@ vclean realclean: clean
 	-del idna.exe  idna.map  idna.pdb  test.exe     test.pdb
 	-rd $(OBJ_DIR)
 
+help:
+	@echo Run one of the following commands:
+	@echo
+	@echo nmake -f Makefile.vc6 VAR=VAL
+	@echo nmake -f Makefile.vc6 install VAR=VAL
+	@echo
```

Comment:
I'd rather make these indented:
```
	@echo "  nmake -f Makefile.vc6 VAR=VAL"
	@echo "  nmake -f Makefile.vc6 install VAR=VAL"
```
I tested with `cmd.exe`, `powershell.exe` and `4nt.exe` as `COMSPEC`. And the extra `"` don't show up.

Reply:
😆 I tried too. I have no idea how to get indentation this way...
Maybe escape the spaces? `\ \ \ \ \`

Reply:
What do you mean? How is this printed in your shell:
```
help:
	@echo Run one of the following commands:
	@echo "  nmake -f Makefile.vc6 VAR=VAL"
	@echo "  nmake -f Makefile.vc6 install VAR=VAL"
	@echo
	@echo where VAR can be of the following:
	@echo "  USE_DEBUG       (default: $(USE_DEBUG))"
	@echo "  USE_DEF_FILE    (default: $(USE_DEF_FILE))"
	@echo "  USE_IP2LOCATION (default: $(USE_IP2LOCATION))"
	@echo "  USE_LUA         (default: $(USE_LUA))"
	@echo "  LUAJIT_ROOT     (default: '$(LUAJIT_ROOT)')"
	@echo "  WANT_64_BITS    (default: check)"
	@echo "  LIB_TARGET      (default: '$(LIB_TARGET)')"
	@echo "  BIN_TARGET      (default: '$(BIN_TARGET)')"
```
I get perfect result:
```
Run one of the following commands:
  nmake -f Makefile.vc6 VAR=VAL
  nmake -f Makefile.vc6 install VAR=VAL

where VAR can be of the following:
  USE_DEBUG       (default: 0)
  USE_DEF_FILE    (default: 1)
  USE_IP2LOCATION (default: 1)
  USE_LUA         (default: 0)
  LUAJIT_ROOT     (default: 'f:\MingW32\src\Parsers\LuaJIT')
  WANT_64_BITS    (default: check)
  LIB_TARGET      (default: 'f:\gv\VC_2017\lib')
  BIN_TARGET      (default: 'f:\gv\VC_2017\bin')
```

Reply:
oh okay. This works. I didn't work for me. So I gave up.

Reply:
Okay. What about this:
```
help:
	@type <<
Run one of the following commands:
  nmake -f Makefile.vc6 VAR=VAL
  nmake -f Makefile.vc6 install VAR=VAL

where VAR can be of the following:
  USE_DEBUG       (default: $(USE_DEBUG))
  USE_DEF_FILE    (default: $(USE_DEF_FILE))
  USE_IP2LOCATION (default: $(USE_IP2LOCATION))
  USE_LUA         (default: $(USE_LUA))
  LUAJIT_ROOT     (default: '$(LUAJIT_ROOT)')
  WANT_64_BITS    (default: check)
  LIB_TARGET      (default: '$(LIB_TARGET)')
  BIN_TARGET      (default: '$(BIN_TARGET)')
<<
```

Reply:
yes, this is even better.